intents-operator/templates/watcher-clusterrole.yaml

```diff
@@ -15,6 +15,29 @@ rules:
       - list
       - watch
       - patch
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs:
+      - create
+      - get
+      - list
+      - patch
+      - update
+      - watch
+  - apiGroups:
```

Comment:
Does this work if this resource is not defined in the cluster? e.g. when Istio is not installed